en/tutorials-and-examples/cms/database.rst

```diff
@@ -123,7 +123,7 @@ Database Configuration
 ======================
 
 Next, let's tell CakePHP where our database is and how to connect to it. Replace
-the values in the ``Datasources.default`` array in your **config/app_local.php** file
+the values in the ``Datasources.default`` array in your **config/app.php** file
```

Comment:
`app_local.php is the correct file: `https://github.com/cakephp/app/blob/master/config/app_local.example.php#L4

Reply:
Oh, my bad.